src/ManifoldsBase.jl

```diff
@@ -1,89 +1,74 @@
 module ManifoldsBase
 
-import Base: isapprox,
-    exp,
-    log,
-    convert,
-    copyto!,
-    angle,
-    eltype,
-    similar,
-    +,
-    -,
-    *
-import LinearAlgebra: dot,
-    norm,
-    det,
-    cross,
-    I,
-    UniformScaling,
-    Diagonal
+import Base: isapprox, exp, log, convert, copyto!, angle, eltype, similar, +, -, *
+import LinearAlgebra: dot, norm, det, cross, I, UniformScaling, Diagonal
 
 import Markdown: @doc_str
 using LinearAlgebra
 
 """
     Manifold
 
-A manifold type. The `Manifold` is used to dispatch to different exponential
-and logarithmic maps as well as other function on manifold.
+A manifold type. The `Manifold` is used to dispatch to different
+[`exp`](@ref)onential and [`log`](@ref)arithmic maps as well as other functions
+on the manifold.
```

Comment:
I think it would be nicer to turn this sentence around:

```
A manifold the. The `Manifold` is used to dispatch to different functions on a manifold.
It should usually be the first argument. Examples are the , [`exp`](@ref)onential and
[`log`](@ref)arithmic maps as well as more general functions that are built on them
like the [`geodesic`](@ref).
```